guides/INSTALLING.md

```diff
@@ -6,15 +6,35 @@ As listed step-by-step in the [Getting Started](https://github.com/ONSdigital/dp
 
 ## Prerequisites
 
-* [Java 8 JDK (OpenJDK)](https://openjdk.java.net/install/)
+In the below, the installation of each app is typically one of:
+- use the `brew` command where provided, or
+- use the link to the website to follow the installation instructions, or
+- follow the link to the Github repo, where you should clone the repo and follow the instructions in the `README.md` file to install/run (within the repo directory)
+
+**Note:** when indicating a command that should be run in your terminal, we use the `$` prefix to indicate your shell prompt.
+
+---
+* [Java 8 JDK (OpenJDK)](https://openjdk.java.net/install/) - install via
+  - `$ brew cask install adoptopenjdk8`
 * [Maven](https://maven.apache.org/)
-* [Docker](https://www.docker.com/get-started)
-* [Cypher Shell](https://neo4j.com/docs/operations-manual/current/tools/cypher-shell/) - installed with `brew install cypher-shell`
-* [Node.js and npm](https://nodejs.org/en/), known working versions:
-  - Node.js version 10.15.3
-  - npm version 6.9.0
+  - `$ brew install maven`
+* [Docker](https://www.docker.com/get-started) and **Docker Compose**
+
+  - `$ brew cask install docker`
+  - `$ brew install docker-compose`
+
+* [Cypher Shell](https://neo4j.com/docs/operations-manual/current/tools/cypher-shell/)
+  - `$ brew install cypher-shell`
+* [Node.js and npm](https://nodejs.org/en/), install the LTS version with:
+
+  - `$ brew install node@12`
+  
```

Comment:
I have just uninstalled node and installed nvm following these instructions https://github.com/nvm-sh/nvm#installing-and-updating and used nvm to install a specific version of node. It was because I run into some problems running one of the applications with the version of node I had, so it might be worth noting that nvm can be used for node installation and to manage versions.

Reply:
Thanks.
As discussed elsewhere, this process will be put into a new document.